apis/kubedb/v1alpha1/postgres_version_types.go

```diff
@@ -0,0 +1,59 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	ResourceCodePostgresVersion     = "pgversion"
+	ResourceKindPostgresVersion     = "PostgresVersion"
+	ResourceSingularPostgresVersion = "postgresversion"
+	ResourcePluralPostgresVersion   = "postgresversions"
+)
+
+// +genclient
+// +k8s:openapi-gen=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PostgresVersion defines a Postgres database version.
+type PostgresVersion struct {
+	metav1.TypeMeta   `json:",inline,omitempty"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              PostgresSpec   `json:"spec,omitempty"`
+	Status            PostgresStatus `json:"status,omitempty"`
+}
+
+// PostgresVersionSpec is the spec for postgres version
+type PostgresVersionSpec struct {
+	// Version
+	Version string `json:"version,omitempty"`
+	// Database Image
+	DB *PostgresVersionDatabase `json:"db,omitempty"`
```

Comment:
Not a pointer, since it is a required field.

Reply:
Is this also the case for Exporter and Tools?

Reply:
Yes.